code/datums/mind.dm

```diff
@@ -1215,6 +1228,47 @@
 				log_admin("[key_name(usr)] has automatically forged objectives for [key_name(current)]")
 				message_admins("[key_name_admin(usr)] has automatically forged objectives for [key_name_admin(current)]")
 
+	else if(href_list["brother"])
+		switch(href_list["brother"])
+			if("clear")
+				if(has_antag_datum(/datum/antagonist/brother))
+					to_chat(src, "<span class='warning'><font size=3><b>You have been brainwashed! You are no longer a blood brother!</b></font></span>")
+					remove_antag_datum(/datum/antagonist/brother)
+					log_admin("[key_name(usr)] has de-brothered [key_name(current)]")
+					message_admins("[key_name_admin(usr)] has de-brothered [key_name_admin(current)]")
+
+			if("brother")
+				if(!has_antag_datum(/datum/antagonist/brother))
+					var/list/candidates = list()
+					for(var/mob/living/L in GLOB.player_list)
+						if(!L.mind || L.mind == current || L.mind.has_antag_datum(/datum/antagonist/brother))
+							continue
+						
+						candidates[L.mind.name] = L.mind
+					/// We pick from the list we just generated who will be the associate antag.
```

Comment:
Dont triple-slash inline code. Thats for var declarations on the class level, not inside a proc